src/app.py

```diff
@@ -78,7 +61,24 @@ def uploadHandler():
         print(error)
         return Response(error.__str__(), status=500)
 
-
+@app.route('/api/mapDateRangeToSemesterPart', methods=['POST'])
+def map_date_range_to_semester_part_handler():
```

Comment:
Maybe sending semesterparts as a list of objects could work as well? Seems like it works

Reply:
I believe it works too. Going to resolve for now.